lib/api/v1/user-controller.js

```diff
@@ -2,14 +2,15 @@
 
 var express = require('express');
 var User = require('../../../models/user');
+var UserPermissions = require('../../../shared/user.js');
 var mailer = require('../../mailer');
 var _ = require('underscore');
 
 module.exports = function(app, user) {
   app = app || express();
-  user = user || require('../authorization')(app);
   var auth = require('../authentication')(app);
   var password = require('../reset-password')(app, auth);
+  user = user || require('../authorization')(app);
```

Comment:
If it is significant that this gets `require`d after the others, there should be a comment saying so.